mcp-server/OSGI_DIAGNOSTICS.md

```diff
@@ -0,0 +1,228 @@
+# OSGi Bundle Diagnostics for AEM/Sling
```

Comment:
I would keep this document out of the PR for now. It's very verbose and will become a maintenance burden